File: Application/ARQuizServer/entities/Competition.php
<?php
/**
 * Created by PhpStorm.
 * User: Martin
 * Date: 13.2.2017
 * Time: 21:24
 */

namespace entities;


class Competition
{
    private $id;
    private $name;
    private $ownerId;
    private $created;
    private $description;

    function __construct($id, $name, $ownerId, $created, $description)
    {
        $this->id = $id;
        $this->name = $name;
        $this->ownerId = $ownerId;
        $this->created = $created;
        $this->description = $description;
    }

    public function getId()
    {
        return $this->id;
    }

    public function getName()
    {
        return $this->name;
    }

    public function getOwnerId()
    {
        return $this->ownerId;
    }

    public function getCreated()
    {
        return $this->created;
    }

    public function getDescription()
    {
        return $this->description;
    }
}

File: Application/ARQuiz/app/src/main/java/com/lepko/martin/arquiz/Entities/Competition.java
package com.lepko.martin.arquiz.Entities;

/**
 * Created by Martin on 6.3.2017.
 */

public class Competition {

    private int id;
    private String name;
    private int ownerId;
    private String created;
    private String description;

    public Competition(int id, String name, int ownerId, String created, String description) {
        this.id = id;
        this.name = name;
        this.ownerId = ownerId;
        this.created = created;
        this.description = description;
    }


    public String getDescription() {
        return description;
    }

    public String getCreated() {
        return created;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }

    public int getOwnerId() {
        return ownerId;
    }
}
